### PR TITLE
fixing python not found issue for task LF, LFV does not run

### PR DESCRIPTION
**Pull request by jzilly**
Python is not found when typing:
docker-compose -f docker-compose-lf.yml pull && \
docker-compose -f docker-compose-lf.yml up 

The change now uses one of the conda python versions installed in the docker container.

### CODE REVIEW

File: docker-compose-lf.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
         volumes:
         - .:/workspace/agent/
         image: duckietown/gym-duckietown-agent:gpu
-        command: /usr/local/bin/python agent/agent.py --no-render
+        command: /opt/conda/bin/python agent/agent.py --no-render
```
Comment (manfreddiaz) [inline docker-compose-lf.yml:20]: @julian121266 why the change to a `conda`-oriented installation?

Comment (jzilly) [reply docker-compose-lf.yml:20]: Not sure there is not another solution, but without this change I get:

Recreating gym-duckietown-agent_gym-duckietown-agent_1  ... error

ERROR: for gym-duckietown-agent_gym-duckietown-agent_1  Cannot start service gym-duckietown-agent: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/local/bin/python\": stat /usr/local/bin/python: no such file or directory": unknown

ERROR: for gym-duckietown-agent  Cannot start service gym-duckietown-agent: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/usr/local/bin/python\": stat /usr/local/bin/python: no such file or directory": unknown
ERROR: Encountered errors while bringing up the project.

Comment (manfreddiaz) [reply docker-compose-lf.yml:20]: Why not just `python`? I think that's one of the advantages of the containerization.

Comment (jzilly) [reply docker-compose-lf.yml:20]: Sure. Can you add it?

**Comment by manfreddiaz** [comment]
Already fixed in a different branch. Thanks, @julian121266.
